multilinear_extensions/src/test.rs

```diff
@@ -31,31 +31,6 @@ fn test_virtual_polynomial_additions() {
     }
 }
 
-#[test]
-fn test_virtual_polynomial_mul_by_mle() {
```

Comment:
Is there a reason why this test is removed? It seems that only `mul_by_mle` is also removed (as it's used just for this test) but it could be kept if we want to keep this test.

Reply:
yes we dont need this functionality anymore, which is just some leftover.